scripts/docgen/doc_source_map.json

```diff
@@ -1,8 +1,9 @@
 {
-    "build": "src/command_modules/vsts-cli-build/vsts/cli/build/_help.py",
```

Comment:
avoid changes in lines which are not touched...
seems like whitespace changed in all lines